package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Node.js client for the new FreshBooks API",
   "main": "freshbooks.js",
   "scripts": {
-    "test": "mocha tests.js"
+    "test": "./node_modules/.bin/mocha tests.js"
```

Comment:
@schuttsm: the `./node_modules/.bin/` shouldn't be necessary, should it?

Reply:
If mocha is installed globally it works, but this works off of a simple "npm install"

I'm getting a "-bash: mocha: command not found" when I run it on my machine